crates/router/src/connector/billwerk.rs

```diff
@@ -634,12 +646,13 @@ impl ConnectorIntegration<api::Execute, types::RefundsData, types::RefundsRespon
         req: &types::RefundsRouterData<api::Execute>,
         _connectors: &settings::Connectors,
     ) -> CustomResult<RequestContent, errors::ConnectorError> {
-        let connector_router_data = billwerk::BillwerkRouterData::try_from((
-            &self.get_currency_unit(),
+        let amount = connector_utils::convert_amount(
```

Comment:
also, can we rename it to refund amount, looks a bit clear. Again optional change

Reply:
Dont require as there is no conflicting names